locals.tf

```
@@ -21,6 +21,7 @@ locals {
       namespace      = "kube-system",
       serviceaccount = "ebs-csi-controller-sa",
     },
+    kubernetes_groups = ["cijio"],
```

```suggestion
    kubernetes_groups = ["ci-jenkins-io"],
```

Full name is easier for reader